clib/src/logging/backend.cc

```diff
@@ -170,19 +172,19 @@ void backend::show_thread_id(bool enable) {
  *
  *  @param[out] buffer  The buffer to fill.
  */
-void backend::_build_header(misc::stringifier& buffer) {
+void backend::_build_header(std::string& buffer) {
   // Build line header.
   if (_show_timestamp == second)
-    buffer << "[" << timestamp::now().to_seconds() << "] ";
+    buffer = fmt::format("[{}] ", timestamp::now().to_seconds());
   else if (_show_timestamp == millisecond)
-    buffer << "[" << timestamp::now().to_mseconds() << "] ";
+    buffer = fmt::format("[{}] ", timestamp::now().to_mseconds());
   else if (_show_timestamp == microsecond)
-    buffer << "[" << timestamp::now().to_useconds() << "] ";
+    buffer = fmt::format("[{}] ", timestamp::now().to_useconds());
   if (_show_pid) {
-    buffer << "[" << getpid() << "] ";
+    buffer += fmt::format("[{}] ", getpid());
   }
   if (_show_thread_id)
-    buffer << "[" << std::this_thread::get_id() << "] ";
+    buffer += fmt::format("[{}] ", std::this_thread::get_id());
 }
```

Comment:
And then you update the string to return

Reply:
will fix